sentry_sdk/integrations/logging.py

```diff
@@ -110,17 +110,32 @@ def _breadcrumb_from_record(record):
     # type: (LogRecord) -> Dict[str, Any]
     return {
         "type": "log",
-        "level": _logging_to_event_level(record.levelname),
+        "level": _logging_to_event_level(record),
         "category": record.name,
         "message": record.message,
         "timestamp": datetime.datetime.utcfromtimestamp(record.created),
         "data": _extra_from_record(record),
     }
 
 
-def _logging_to_event_level(levelname):
-    # type: (str) -> str
-    return {"critical": "fatal"}.get(levelname.lower(), levelname.lower())
+def _logging_to_event_level(record):
+    # type: (LogRecord) -> str
+    to_event_level = {
+        logging.NOTSET: "notset",
+        logging.DEBUG: "debug",
+        logging.INFO: "info",
+        logging.WARN: "warning",  # WARN is same a WARNING
+        logging.WARNING: "warning",
+        logging.ERROR: "error",
+        logging.FATAL: "fatal",
+        logging.CRITICAL: "fatal",  # CRITICAL is same as FATAL
+    }
+
+    event_level = to_event_level.get(record.levelno) or to_event_level.get(
+        DEFAULT_EVENT_LEVEL
+    )
```

Comment:
to preserve some older behaviour, should we do
```suggestion
    event_level = to_event_level.get(record.levelno, recrord.levelname.lower())
```